applications/RANSApplication/python_scripts/formulations/turbulence_models/k_omega_rans_formulation.py

```diff
@@ -0,0 +1,142 @@
+            "coupling_settings":
+            {
+                "relative_tolerance": 1e-3,
+                "absolute_tolerance": 1e-5,
+                "max_iterations": 10
+            },
+            "auxiliar_process_list": [],
+            "echo_level": 0
+        }''')
+
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        super().__init__(
+            model_part,
+            settings,
+            KOmegaKRansFormulation(model_part, settings["turbulent_kinetic_energy_solver_settings"]),
+            KOmegaOmegaRansFormulation(model_part, settings["turbulent_specific_energy_dissipation_rate_solver_settings"]))
+
+    def AddVariables(self):
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(Kratos.DENSITY)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(Kratos.VELOCITY)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(Kratos.MESH_VELOCITY)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(Kratos.NORMAL)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(Kratos.VISCOSITY)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(Kratos.KINEMATIC_VISCOSITY)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(Kratos.TURBULENT_VISCOSITY)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(KratosRANS.RANS_Y_PLUS)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(KratosRANS.TURBULENT_KINETIC_ENERGY)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(KratosRANS.TURBULENT_KINETIC_ENERGY_RATE)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(KratosRANS.TURBULENT_SPECIFIC_ENERGY_DISSIPATION_RATE)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(KratosRANS.TURBULENT_SPECIFIC_ENERGY_DISSIPATION_RATE_2)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(KratosRANS.RANS_AUXILIARY_VARIABLE_1)
+        self.GetBaseModelPart().AddNodalSolutionStepVariable(KratosRANS.RANS_AUXILIARY_VARIABLE_2)
+
+        Kratos.Logger.PrintInfo(self.__class__.__name__, "Added solution step variables.")
+
+    def AddDofs(self):
+        Kratos.VariableUtils().AddDof(KratosRANS.TURBULENT_KINETIC_ENERGY, self.GetBaseModelPart())
+        Kratos.VariableUtils().AddDof(KratosRANS.TURBULENT_SPECIFIC_ENERGY_DISSIPATION_RATE, self.GetBaseModelPart())
+
+        Kratos.Logger.PrintInfo(self.__class__.__name__, "Added solution step dofs.")
+
+    def Initialize(self):
+        model_part = self.GetBaseModelPart()
+        model = model_part.GetModel()
+
+        process_info = model_part.ProcessInfo
+        wall_model_part_name = process_info[KratosRANS.WALL_MODEL_PART_NAME]
+        kappa = process_info[KratosRANS.WALL_VON_KARMAN]
+
+        nut_process = KratosRANS.RansNutKOmegaUpdateProcess(
+                                            model,
+                                            self.GetBaseModelPart().Name,
+                                            1e-12,
```

Comment:
I'd use an auxiliary variable to specify what this `1e-12` is.

Reply:
done :)